dist/main.css

```diff
@@ -5483,3 +5483,582 @@ $blueBG8: #75cbe8;
+
+$blackFC1: #575757;
+$blackFC2: #282834;
+$blackFC3: black;
+$blackFC4: #040404;
+$blackFC5: #4f4d4d;
+$blackFC6: #161616;
+$blackFC7: #818080;
+$blackFC8: #6f6f6f;
+
+$grayFC1: #b4bad9;
+$grayFC2: #6E6E6E;
+$grayFC3: #BDBBBB;
+$grayFC4: #5E656B;
+$grayFC5: #B5C1C8;
+$grayFC6: #3E3E3E;
+$grayFC7: #363636;
+$grayFC8: #4d4d4d;
+$grayFC9: #a1a1a1;
+$grayFC10: #908989;
+$grayFC11: #757a83;
+$grayFC12: #aeaeae;
+$grayFC13: #b3b3b3;
+$grayFC14: #A9A9A9;
+$grayFC15: #a4a7ab;
+$grayFC16: #f0f0f1;
+$greyFC19: #757a83;
+$grayFC17: #e1e1e1;
+$grayFC18: #b7b7b7;
+$grayFC19: #dad5d5;
+$grayFC22: #E6E5E6;
+$grayFC23: #f3efef;
+$grayFC24: #fdfcfc;
+$grayFC25: #626060;
+$grayFC26: #dfe2e2;
+$grayFC27: #f7f6f6;
+$grayFC28: #625a5a;
+$grayFC29: #625a5a;
+$grayFC30: #b5b5b5;
+$grayFC31: #6E6E6E;
+$grayFC32: #4a4a4a;
+$grayFC33: #c7c7c6;
+$grayFC34: #95989c;
+$grayFC34: #807f7f;
+$grayFC35: #c2c2c2;
+$grayFC36: #898989;
+$grayFC37: #d2d2d2;
+$grayFC38: #f7f7f7;
+$grayFC39: #f3f3f3;
+$grayFC40: #959595;
+$grayFC41: #B0B1B2;
+$grayFC42: #7A7A7A;
+$grayFC43: #e2e2e2;
+$greyUP1: #757a83;
+
+$yellowFC1: #ff931e;
+$yellowFC2: #8c6f09;
+
+$orangeFC1: #ad6414;
+$orangeFC2: #f79226;
+$redFC1: #F94040;
+$redFC2: #ff6766;
+$redFC3: #FD4E53;
+
+$greenFC1: #87ca4d;//todo(igor) overriding existing variable !!!
+$BlueFC1: #0d9bad;
+$darkBlueFC1: #2a3747;
+
+$turquoiseFC1: #0a9bad;
+
+$pinkFC1: #d64a7d;
+$pinkFC2: #992f56;
+
+$blueFC1: #225060;
+
+$purpleFC1: #695081;
+//border color
+$greyBC1: #e1e8ed;
+$greyBC2: #CCD6DD;
+$greyBC3: #B5C1C8;
+$greyBC4: #CFCFCF;
+$greyBC5: #eeeff1;
+$greyBC6: #E2E2E2;
+$greyBC7: #ACACAC;
+$greyBC8: #d4d4d4;
+$greyBC9: #e9e9e9;
+$greyBC10: #979c9c;
+$greyBC11: #d0cece;
+$greyBC12: #95989c;
+$greyBC13: #e2e7ec;
+
+$whiteBC1: white;
+$whiteBC2: #E4E4E4;
+
+$redBC1: #F94040;
+$redBC2: #ff6766;
+$redBC3: #e9e3e3;
+$redBC4: #ff9a99;
+
+$greenBC1: #87ca4d;
+$greenBC2: #c9e6b5;
+
+$turquoiseBC1: #0a9bad;
+$turquoiseBC2: #0c8d9d;
+
+$blackBC1: #000;
+
+$pinkBC1: #992f56;
+
+$yellowBC1: #eabb12;
+
+$orangeBC1: #ff931e;
+
+$purpleBC1: #8f6fad;
+//list subject boxs
+//change names
+$blueMath : #6cbfdb;
+$blueMathDone: #3b89a4;
+$blueDoneBorder:#225060;
+$orangeRead: #ffca08;
+$yellowReadDone: #e8b80b;
+$yellowDoneBorder: #8c6f09;
+$pinkWrite: #ff5895;
+$pinkWriteDone: #d64a7d;
+$pinkDoneBorder: #992f56;
+
+
+
+$blueCompleteMath : #66b6d1;
+$orangeCompleteRead: #ecbc0f;
+$pinkCompleteWrite: #dd4b81;
+
+//Header
+$blueBG1: #0d9cae; //used in new app header
+$blueBG2: #0a9bad; //used in blue login button
+$blueBG3: #5794a9;
+$blueBG4: #88b8c9;
+$blueBG5: #6c9fb1;
+$blueBG6: #84CDE8;
+$blueBG7: #75cbe8;
+
+$orangeBG1 : #ffa33f;
+
+$blackFC1: #274854;
+$blackFC2: #5A4A0F;
+$blackFC2: #47202E;
+
+//New Colors for Toefl
+
+$yellowBG6: #f9d41b;
+
+
+$blueBG8: #75cbe8;
+
+
+
+*/
+/* paths*/
+/* fonts size */
+/* full background image with retina */
+/**
+ * Tooltip Styles
+ */
+/* Add this attribute to the element that needs a tooltip */
+[data-tooltip] {
```

Comment:
no.
the part above the comment doesn't look familiar
the part bellow yes. it's new tooltip.scss